packages/tools/u-boot/package.mk

```diff
@@ -32,6 +32,7 @@ case "$PROJECT" in
     PKG_VERSION="v2020.01"
     PKG_SHA256="e981e02592f7b5386e5bc2b8a076aebc47d90b0ec9ae62be4b76daadc083b3ef"
     PKG_URL="https://github.com/u-boot/u-boot/archive/$PKG_VERSION.tar.gz"
+    PKG_PATCH_DIRS="all"
```

Comment:
I would copy the linux package and use `default` instead of `all` for this. NB: Once u-boot 2020.04 ships (eta 6th April) the VIM3 patches are upstream, so we can probably bump u-boot and then you can drop this change.

Reply:
addressed, thanks